contract/Cargo.toml

```diff
@@ -12,7 +12,7 @@ library = []
 
 [dependencies]
 cosmwasm-schema.workspace = true
-cosmwasm-std.workspace = true
+cosmwasm-std = { workspace = true, features = ["cosmwasm_2_2"] }
```

Comment:
Do we know why we need this capability? The current version of the chain doesn't have this capability, and the doc doesn't even have info about it.  cc @gluax

Reply:
Yes, I stated it above, has to be at least cosmwasm 2_1 or the contract tests will no longer work.

Reply:
Aha Got it.. will set it to 2_1 then since our chain already has that.